src/components/navigation/www/navigation.jsx

```diff
@@ -202,21 +202,16 @@ var Navigation = React.createClass({
                             <FormattedMessage id="general.explore" />
                         </a>
                     </li>
-                    <li className="link discuss">
-                        <a href="/discuss">
-                            <FormattedMessage id="general.discuss" />
```

Comment:
If this is the only use of general.discuss, perhaps we should also remove it from the l10n.json file.